assets/js/page-manager.js

```diff
@@ -252,16 +252,15 @@ class PageManager {
             return;
         }
 
-        if (!event.target.dataset.pageOnClick) {
-            return;
+        for (var t = event.target; t && t != event.currentTarget; t = t.parentNode) {
+            if (t.dataset.pageOnClick) {
+                event.preventDefault();
+                const requestedPageId = t.dataset.pageOnClick;
+                const data = this._extractPageDataArgs(t);
+                this.loadPage(requestedPageId, data);
+                break;
```

Comment:
I'm not sure if `break` is needed here, i.e. whether `this.loadPage` returns. Let me know if you know for sure that it doesn't?